Model/src/BoardPackage/BoardSquare.java

```diff
@@ -1,7 +1,93 @@
+    protected String name;
+
+    /**An ArrayList of Players that tracks what players are on this
+     * square.
+     */
+    protected ArrayList<Player> occupiedBy;
+
+    /*****************************************************************
+     * Constructor instantiates the variables and sets the name of the
+     * BoardSquare to what is input as a parameter.
+     *
+     * @param name String that will be the name of the BoardSquare.
+     ****************************************************************/
+    public BoardSquare(String name) {
+        this.name = name;
+        occupiedBy = new ArrayList<Player>(6);
+    }
+
+    /*****************************************************************
+     *The getName method returns the name of the BoardSquare
+     *
+     * @return name The string that is the name of the BoardSquare.
+     ****************************************************************/
+    public String getName() {
+        return name;
+    }
+
+     /****************************************************************
+      * The setName method sets the name of the BoardSquare.
+      *
+      * @param name The name that this BoardSquare is set to.
+      ***************************************************************/
+    public void setName(String name) {
+        this.name = name;
+    }
+
+     /****************************************************************
+      * The getOccupiedBy method returns the Arraylist that is the
+      * current players on this BoardSquare
+      *
+      * @return occupiedBy the list of players on this BoardSquare.
+      ***************************************************************/
+    public ArrayList<Player> getOccupiedBy() {
+        return occupiedBy;
+    }
+
+     /****************************************************************
+      * The setOccupied by sets what players are on this BoardSquare.
+      *
+      * @param occupiedBy the list of players that will be who is
+      *                   occupying this BoardSquare.
+      ***************************************************************/
+    public void setOccupiedBy(ArrayList<Player> occupiedBy) {
+        this.occupiedBy = occupiedBy;
+    }
+
+     /****************************************************************
+      * The addPlayer method adds new player that lands on this
+      * BoardSquare.
+      *
+      * @param p the Player landing on this BoardSquare.
+      ***************************************************************/
+    public  void addPlayer(Player p){
+        occupiedBy.add(p);
+    }
+
+     /****************************************************************
+      * The removePlayer method removes a player who is leaving
+      * this BoardSquare.
+      *
+      * @param p the Player leaving this BoardSquare.
+      ***************************************************************/
+    public void removePlayer(Player p){
+        occupiedBy.remove(p);
+    }
```

Comment:
maybe add an abstract method for the card Description.

Reply:
card Description?

Reply:
i was thinking of a method that returns the card properties or something, just in case. If you don't think is a good idea is fine.

Reply:
I wrote the class as it was described in the UML diagram on Slack. Card description goes with the Card Class.

Reply:
I think we leave this out for now, Lets build what we have in the UML diagram first, then we can create issues to modify things once its all created.

Reply:
agree. Sorry about that